js/index.js

```diff
@@ -0,0 +1,30 @@
+// listen to form submit event and make sure page doesn't refresh
+// grab value out of input field
+// make a decision: is that city NYC, if it is, change background, else - if that city is LA, change bg to LA
+$(document).ready(function() {
+
+	$('form').submit(addEntry);
+
+	function addEntry(event) {
+		event.preventDefault();
+		var userInput = $('#city-type').val();
+		console.log(userInput)
+		if (userInput==='austin' || userInput==='Austin') {
+			 $('body').addClass('austin');
```

Comment:
instead of `.addClass` here you want totally rewrite what's in the `<body class="">` value. Otherwise you could end up with `<body class="austin sf la sydney nyc">`

So instead, I recommend just doing 
```
$('body').attr('class', 'austin');
```